src/main/resources/db/migration/V1__FF4J_schema_from_library.sql

```diff
@@ -0,0 +1,54 @@
+-- Main Table to store Features
```

Comment:
add leading zeros in version in file name

Reply:
oh, sillies mes
> I meant the migration file name